06_gpu_and_ml/llm-serving/vllm_inference.py

```diff
@@ -35,32 +35,34 @@
 vllm_image = (
     modal.Image.from_registry("nvidia/cuda:12.8.0-devel-ubuntu22.04", add_python="3.12")
     .uv_pip_install(
-        "vllm==0.10.1.1",
-        "huggingface_hub[hf_transfer]==0.34.4",
-        "flashinfer-python==0.2.8",
-        "torch==2.7.1",
+        "vllm==0.10.2",
+        "huggingface_hub[hf_transfer]==0.35.0",
+        "flashinfer-python==0.3.1",
+        "torch==2.8.0",
     )
     .env({"HF_HUB_ENABLE_HF_TRANSFER": "1"})  # faster model transfers
 )
 
 # ## Download the model weights
 
-# We'll be running a pretrained foundation model -- Meta's LLaMA 3.1 8B
-# in the Instruct variant that's trained to chat and follow instructions.
+# We'll be running a pretrained foundation model -- Qwen's Qwen3-8B,
+# it is trained with thinking capabilities. This means the model will
+# use its reasoning abilities to enhance the quality of generated responses.
 
 # Model parameters are often quantized to a lower precision during training
 # than they are run at during inference.
-# We'll use an eight bit floating point quantization from Neural Magic/Red Hat.
+# We'll use an eight bit floating point quantization.
```

Comment:
This sentence no longer flows as cleanly since we don't have a source to point to. Remove?